js/ui/overview.js

```diff
@@ -432,9 +432,18 @@ var Overview = class {
         // To avoid updating the position and size of the workspaces
         // we just hide the overview. The positions will be updated
         // when it is next shown.
+        const visible = this.visible;
         this.hide();
 
         this._relayoutNoHide();
+
+        // The previous hide call will make *NotVisible* the apps icons
+        // in the desktop. If it was visible before this relayout we should
+        // show again to avoid a desktop with disabled icons and no apps opened
+        if (visible) {
+            this.visible = false;
```

Comment:
Why do we need to set this to `false` here?

Reply:
This is set to false here because the `showApps` does nothing if `visible` is true.